nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/processors/script/ExecuteScriptGroovyTest.groovy

```diff
@@ -150,7 +142,10 @@ class ExecuteScriptGroovyTest extends BaseScriptTest {
         }
     }
 
-    @Ignore("This test fails intermittently when the serial execution happens faster than pooled")
+    @EnabledIfSystemProperty(
+            named = "nifi.test.unstable",
+            matches = "true",
+            disabledReason = "This test fails intermittently when the serial execution happens faster than pooled")
```

Comment:
Recommend removing this test method. With Git history, if others are interested in improving it, the previous approach is available.  As it stands, this seems equivalent to commented code, and thus not useful.

Reply:
Fair point.